src/commands/app/add/event.js

```diff
@@ -29,27 +31,63 @@ class AddEventCommand extends AddCommand {
     if (entries.length > 1) {
       this.error('Please use the \'-e\' flag to specify to which implementation you want to add events to.')
     }
+
     const configName = entries[0][0]
     const config = entries[0][1]
     const actionFolder = path.relative(config.root, config.actions.src)
     const configData = this.getRuntimeManifestConfigFile(configName)
```

Comment:
nitpick: should we rename`configData` => `runtimeData` (to contrasts with eventsData below)

Reply:
does not matter since both a local to the respective methods.

Reply:
done

Reply:
thanks